includes/RemovePIIJob.php

```diff
@@ -64,7 +61,7 @@ public function run() {
 			return false;
 		}
 
-		$dbw = wfGetDB( DB_PRIMARY, [], $this->database );
+		$dbw = wfGetDB( DB_PRIMARY );
```

Comment:
since you already have a LBFactory, for consistency would be nice to use `$lbFactory->getConnection( DB_PRIMARY )`

Reply:
Done.